docs/data/data-grid/events/events.md

```diff
@@ -2,36 +2,72 @@
 title: Data Grid - Events
 ---
 
-# Data grid - Events [<span class="plan-pro"></span>](https://mui.com/store/items/mui-x-pro/)
+# Data grid - Events
 
-<p class="description">The data grid emits events that can be subscribed to attach custom behavior.</p>
+<p class="description">Subscribe to the events emitted by the data grid to trigger custom behavior.</p>
```

Comment:
Rephrasing slightly to frame it in terms what the developer can do with the feature.

```suggestion
<p class="description">You can subscribe to the events emitted by the data grid to trigger custom behavior.</p>
```